rules/crowdstrike_rules/crowdstrike_dns_request.py

```diff
@@ -1,17 +1,28 @@
+from panther_base_helpers import get_crowdstrike_field
+
 # baddomain.com is present for testing purposes. Add domains you wish to be alerted on to this list
 DENYLIST = ["baddomain.com"]
 
 
 def rule(event):
-    if event.get("DomainName") in DENYLIST:
+    # We need to run either for Crowdstrike.DnsRequest or for DnsRequest.FDREvent of 'DnsRequest'
+    # type. Crowdstrike.DnsRequest is covered because of the association with the type
+    if (
+        event.get("p_log_type") == "Crowdstrike.FDREvent"
```

Comment:
I don't think you need this condition, if `Crowdstrike.DNSRequest` and `Crowdstrike.FDREvent` are the only log types associated with this detection.

Reply:
Shouldn't we only for for `fdr_event_type == 'DnsRequest'` only though?

Reply:
This is done too. Resolving as it's outdated.

Reply:
This doesn't seem correct. 

If I just have `fdr_event_type == 'DnsRequest'` it wouldn't match with (old) `Crowdstrike.DNSRequest` as those events do not have that `fdr_event_type` field.

Reply:
Ah you're right, sorry for that. There are other ways to write this condition, but I think the explicit check denotes that we're taking FDREvent into account here!